src/web/releases.rs

```diff
@@ -668,20 +669,58 @@ impl_webpage! {
 
 pub fn activity_handler(req: &mut Request) -> IronResult<Response> {
     let mut conn = extension!(req, Pool).get()?;
-    let activity_data: Value = ctry!(
+
+    let data: Vec<(NaiveDate, i64, i64)> = ctry!(
         req,
         conn.query(
-            "SELECT value FROM config WHERE name = 'release_activity'",
-            &[]
-        ),
+            "
+            WITH dates AS (
+                -- we need this series to we have also days in the statistic that don't have any releases
+                SELECT generate_series( 
+                        CURRENT_DATE - interval '30 days',
+                        CURRENT_DATE - interval '1 day',
+                        '1 day'::interval
+                    )::date AS date_
+            ), 
+            release_stats AS (
+                SELECT
+                    release_time::date AS date_,
+                    count(*) AS counts,
+                    sum(CASE WHEN is_library = TRUE AND build_status = FALSE THEN 1 ELSE 0 END) AS failures
```

Comment:
I think you can cast the boolean to an int.l instead of needing a case.

Reply:
I typically find these `CASE WHEN` statements better readable, 
but I'll change it if you prefer the `CAST`. 

(performance wise I doubt it makes a difference)